### PR TITLE
Update utils.js

### PR DESCRIPTION
**Pull request by Turtus**
for anyone who do need segwit inputs actualized https://github.com/bitcoinjs/coinselect/issues/56

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 // baseline estimates, used to improve performance
 var TX_EMPTY_SIZE = 4 + 1 + 1 + 4
 var TX_INPUT_BASE = 32 + 4 + 1 + 4
+var TX_INPUT_SEGWIT = 34
```
Comment (jaonoctus) [inline utils.js:4]: @Turtus Why 34?

btw, a transaction input will always have at least 41 bytes (`TX_INPUT_BASE = 32 + 4 + 1 + 4`).

- 32 bytes for the [previous txid hash](https://learnmeabitcoin.com/technical/txid)
- 4 bytes for the ([vout](https://learnmeabitcoin.com/technical/vout))
- 1 byte for the scriptSigSize [varint](https://learnmeabitcoin.com/technical/varint)
- 4 bytes for the sequence

https://learnmeabitcoin.com/technical/input

**Review by jaonoctus — CHANGES_REQUESTED** [review]

**Comment by jaonoctus** [comment]
The best option for this workaround seems to be **not change anything on the algo**, and add for each P2WPKH (native segwit singlesig) input and output, `script.length = 27` instead.

```
function inputBytes (input) {
  if (typeof input.isSegwit !== 'undefined' && input.isSegwit) return TX_INPUT_SEGWIT;
  return TX_INPUT_BASE + (input.script ? input.script.length : TX_INPUT_PUBKEYHASH)
}

function outputBytes (output) {
  return TX_OUTPUT_BASE + (output.script ? output.script.length : TX_OUTPUT_PUBKEYHASH)
}
```

I'm not sure about P2WSH (multisig) or P2TR (taproot) tho 🤔

**Comment by Turtus** [comment]
dno problem - ive already using script "with predefined" length not to patch the lib, but actual version is sooo not supporting segwit that anything will be improvement)

**Comment by jaonoctus** [comment]
@Turtus which script is that?

I agree :P

**Comment by Overtorment** [comment]
fyi, this is how its approached in bluewallet: https://github.com/BlueWallet/BlueWallet/pull/3810

**Pull request closed without merging**